cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -349,29 +352,42 @@ SwiftGenerator::writeDocSummary(IceInternal::Output& out, const ContainedPtr& p)
         return;
     }
 
+    bool hasStarted = false;
```

Comment:
If a doc comment didn't have an overview, but did document other things, we would emit a leading line:
```
///
/// - Parameter hello: something.
```
Because we were always assuming an overview was present. This bool lets us only write the separating line when necessary.